Testing/TemplateTestCase.m

```diff
@@ -0,0 +1,61 @@
+%% Template Test Case Setup

```

Comment:
I'm not sure what these `^M` things are but it [appears to be because you're mixing CRLF (Windows, `\r\n`) and LF (Unix, `\n`)](https://github.com/atom/atom/issues/3105#issuecomment-304492881). You should probably just make the whole file LF.

Reply:
Converted all files to LF, as well as set my default to LF from now on to match the other files.